skimage/measure/tests/test_segmentation_metrics.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+
+from skimage.measure import (compare_adapted_rand_error,
+                             compare_split_variation_of_information,
+                             compare_variation_of_information)
+
+from skimage._shared.testing import assert_equal, assert_almost_equal
+
+def test_vi():
+    im_true = np.array([1, 2, 3, 4])
+    im_test = np.array([1, 1, 8, 8])
+    assert_equal(compare_variation_of_information(im_true, im_test), -1)
```

Comment:
I'm not sure what's happening here, but VI is always non-negative.

Reply:
both vi and split vi are returning negative numbers every time. I'm not sure whats going on either